numbers/search-available.php

```diff
@@ -9,8 +9,17 @@
 $client = new \Vonage\Client($basic);
 
 /** @var IterableAPICollection $response */
+/** Note: be careful when specifying "NUMBER_SEARCH_CRITERIA" environment variable: 
```

Comment:
Can we revert the changes made in this file and handle it all in #60 ?

Reply:
I'm afraid I have made a monumental mess of these pull requests.
Do you want to just close these two and I will create a new pull request with a single, merged commit that has fixes for both Issue #51 and Issue #52?  I will merge them into my forked master branch as a squash commit and then create a pull request for you.  How does that sound?  Thank you for being patient.